src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
On top of that, it is not consistent with the super implementation of selectionForIndex(). I was having different expectations from this method when I was playing with it and wasted some time figuring out that it is accessing the same data as the parent class but in a different way.